### ISSUE TITLE
Possible format error retrieving secret 

### ISSUE THREAD

**Issue by dwightjack** [google-github-actions/get-secretmanager-secrets#262]
### TL;DR

The secret retrieved by this action seems to be corrupted or different from the one returned by the `gcloud secrets versions access` command.

### Expected behavior

The secret should work

### Observed behavior

The secret is not valid

### Action YAML

I cannot paste the action because it's a private repository, anyway here are the relevant parts:

#### Working

```yaml
- name: Setup deploy variables
id: secrets
run: |
  ACCESS_KEY_ID=$(gcloud secrets versions access 1 --secret=access_key_id)
  SECRET_ACCESS_KEY=$(gcloud secrets versions access 1 --secret=secret_access_key)
  echo "::add-mask::$ACCESS_KEY_ID"
  echo "::add-mask::$SECRET_ACCESS_KEY"
  echo "SECRET_ACCESS_KEY=$SECRET_ACCESS_KEY" >> $GITHUB_OUTPUT
  echo "ACCESS_KEY_ID=$ACCESS_KEY_ID" >> $GITHUB_OUTPUT
- name: Deploy 
  run: |
    # requires SECRET_ACCESS_KEY and ACCESS_KEY_ID
    pnpm run deploy
  env:
    SECRET_ACCESS_KEY: ${{ steps.secrets.outputs.SECRET_ACCESS_KEY }}
    ACCESS_KEY_ID: ${{ steps.secrets.outputs.ACCESS_KEY_ID }}
```

#### Not working

```yaml
- name: Retrieve required env variables
  id: secrets
  uses: google-github-actions/get-secretmanager-secrets@v1
  with:
    secrets: |-
      access_key_id:projects/000000000/secrets/access_key_id/versions/1
      secret_access_key:projects/000000000/secrets/secret_access_key/versions/1
- name: Deploy 
  run: |
    # requires SECRET_ACCESS_KEY and ACCESS_KEY_ID
    pnpm run deploy
  env:
    SECRET_ACCESS_KEY: ${{ steps.secrets.outputs.secret_access_key }}
    ACCESS_KEY_ID: ${{ steps.secrets.outputs.access_key_id }}
```

### Log output

Except for the error in the deploy command (reporting that the something was wrong with the authorizations), the only visible difference is that the env variables set with the output generated by the action are separated by a blank line:

#### When it works

```text
GCP_PROJECT: XXXXXX
SECRET_ACCESS_KEY: ***
ACCESS_KEY_ID: ***
```

#### When it does not work
```
GCP_PROJECT: XXXXXX
SECRET_ACCESS_KEY: ***

ACCESS_KEY_ID: ***
```

**Comment by dwightjack** [comment]
@sethvargo thanks for the reply. I suspected that was the issue, I will check the secrets again. 
About AWS, those secrets are consumed by a cli tool so I don't have direct access to the infrastructure, anyway I'll pass the suggestion to the team. Thanks 🙇
